src/nixos/machines/tupac/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+# shellcheck shell=sh # POSIX
+
+# Invokation: PATH_TO_THIS_REPO $ nix-shell -p zfs --run ./src/nixos/machines/tupac/deploy.sh
+
+export LC_ALL=C
```

Comment:
unclear if this is necessary for the script's operations

Reply:
> unclear if this is necessary for the script's operations -- @TanvirOnGH (https://github.com/NiXium-org/NiXium/pull/74#discussion_r1660036148)

Surprisingly it turned out to be critical as disko will deploy a very disfunctional system without it, i plan on fixing that upstream later.